scripts/run-dev.sh

```diff
@@ -7,4 +7,4 @@ cd ../
 npm run build
 
 # move static code to bower components for examples
```

Comment:
Will you update this comment to not say "bower", just for consistency?

Reply:
Good catch, updated and also attended to the ignore patterns. All should be fine now.